make/modules/java.base/Lib.gmk

```diff
@@ -52,7 +52,8 @@ $(eval $(call SetupJdkLibrary, BUILD_LIBNET, \
     DISABLED_WARNINGS_microsoft_ResolverConfigurationImpl.c := 4996, \
     LDFLAGS := $(LDFLAGS_JDKLIB) \
         $(call SET_SHARED_LIBRARY_ORIGIN), \
-    LDFLAGS_windows := -delayload:secur32.dll -delayload:iphlpapi.dll, \
+    LDFLAGS_windows := -delayload:secur32.dll -delayload:iphlpapi.dll \
```

Comment:
Wonder if this could be `LDFLAGS_microsoft_windows` to emphasize the compiler that this applies to

Reply:
I wouldn't mind that if you think it's only reasonable to support this with Visual Studio.

Reply:
it's useful for VS and clang, but not supported by GCC (yet?). Since neither GCC nor Clang is supported now, I'd rather keep it.

Reply:
@TheShermanTanker I understand where you are coming from, but as a matter of fact we still only support building on Windows with Visual Studio, and as long as we do, we only need to specify "windows" in situations like these.